src/alias/index.ts

```diff
@@ -0,0 +1,77 @@
+import { ConnectionREST } from '../index.js';
+import { WeaviateAlias, WeaviateAliasResponse } from '../openapi/types.js';
+import { Alias, CreateAliasInput, UpdateAliasInput } from './types.js';
+
+export interface Aliases {
+  /**
+   * Create alias for a collection.
+   *
+   * The collection must exist prior to aliasing it.
+   * One alias cannot be created for multiple collections simultaneously.
+   *
+   * @param {string} [opt.collection] Original collection name.
+   * @param {string} [opt.alias] Alias for collection.
+   * @returns {Promise<void>} Awaitable promise.
+   * */
+  create: (opt: CreateAliasInput) => Promise<void>;
```

Comment:
Imo, since `opt` here is required (doesn't allow undefined `opt?`) and `collection` + `alias` inside the type are also not able to `undefined`, it should be named `args: CreateAliasArgs` to align with other areas of the codebase, e.g. `Backup.cancel`, wdyt?

In addition, required params in TSdoc should not have the `[]` wrapping. I think there are issues elsewhere in the codebase outside of this PR with the consistency of this but we should make sure new code follows it regardless 😁